src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="section">
+      <div class="calendar calendar--start-day-tue calendar--month-length-30">
```

Comment:
checklist #2 (calendar--start-day--tue), same for month length

Reply:
Fix BEM naming